charts/ingress-apisix/values.yaml

```diff
@@ -0,0 +1,66 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Default values for ingress-apisix.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
```

Comment:
I think this chart has too many configurable options for users, such as `affinity`, `label`, `annotation`, `toleration`, etc, which leads to too many configurations of values.yaml, but most of them are not help to controller's logic  It is rarely used even when I managing the resources of a large cluster).

I think the number of options for an easy-to-use chart should be just enough, and then gradually evolve, the more the configuration, the easier it is to make mistakes, whether it is for the user or the maintenance person.

Reply:
Most of them are just created by `helm create`, it seems these are not so useful, i will clean some of them.

Reply:
Got it